src/test/java/com/foundation/salesforce/steps/ContactAcceptanceSteps.java

```diff
@@ -0,0 +1,268 @@
+    @Then("User get a {string} status code with firstName, email")
+    public void userGetAStatusCodeWithFirstNameEmail(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to let user specifies new contact with first name, email and title.
+     *
+     * @param inputContent
+     */
+    @Given("User specifies new contact with firstName, email, title")
+    public void userSpecifiesNewContactWithFirstNameEmailTitle(Map<String, String> inputContent) {
+        RestClientApi.getInstance().buildSpec(inputContent);
+    }
+
+    /**
+     * Method to let a send a request to contact with first name , email and title.
+     */
+    @When("User send de request post to contact endpoint with firstName, email, title")
+    public void userSendDeRequestPostToContactEndpointWithFirstNameEmailTitle() {
+        response = RestClientApi.getInstance().post(EndPoints.CONTACT_ENDPOINT);
+    }
+
+    /**
+     * Method to let get the statutus with the first name, email and title.
+     *
+     * @param statusCode
+     */
+    @Then("User get a {string} status code with firstName, email, title")
+    public void userGetAStatusCodeWithFirstNameEmailTitle(String statusCode) {
+        Assert.assertEquals(response.getStatusCode(), Integer.parseInt(statusCode));
+    }
+
+    /**
+     * Method to let delete a user existing..
+     */
+    @When("User deletes an existing contact by Id")
+    public void userDeletesAnExistingContactById() {
+        response = requestManager.delete(EndPoints.CONTACT_ENDPOINT + "/" + "00Q3i000002MRlqEA1");
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le Delete a contact by id.
+     *
+     * @param arg0
+     */
+    @When("User deletes a Contact by Id (.*)")
+    public void userDeletesAContactByIdQIMRlqEAG(int arg0) {
+        response = requestManager.delete(EndPoints.CONTACT_ENDPOINT + "/" + arg0);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to led find an existing contact by id.
+     */
+    @When("User finds an existing Contact by Id")
+    public void userFindsAnExistingContactById() {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT + "/" + "");
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to led find a contact by id.
+     *
+     * @param arg0
+     */
+    @When("User finds a contact by Id (.*)")
+    public void userFindsAContactByIdQIMKLeEA(int arg0) {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT + "/" + arg0);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le retrieves the summary for a contact.
+     */
+    @When("User retrieves the summary for Contact")
+    public void userRetrievesTheSummaryForContact() {
+        response = requestManager.get(EndPoints.CONTACT_ENDPOINT);
+        response.prettyPrint();
+    }
+
+    /**
+     * Method to le verify status code.
+     *
+     * @param statusCode
```

Comment:
Please use an appropriate name for this input parameter.

Please, place descriptions for input parameters.